lib/components/user/monitored-trip/trip-status-rendering-strategies/upcoming-trip-renderer.js

```diff
@@ -33,16 +34,32 @@ export default function upcomingTripRenderer ({
     monitoredTrip.leadTimeInMinutes,
     'minutes'
   )
-  data.bodyText =
-    `Trip is due to begin at ${tripStartMoment.format(timeFormat)}. (realtime monitoring will begin at ${monitoringStartMoment.format(timeFormat)})`
+  data.bodyText = (
+    <FormattedMessage
+      id='components.TripStatusRenderers.upcoming.tripBegins'
+      values={{
```

Comment:
Since we decided to inline the date/time formats, do the same for this message (update the message in the language files), remove the moment variables.